### ISSUE TITLE
Any reason to think that Spotify would even know that a track was being download if played at normal speed?

### ISSUE THREAD

**Issue by dtronvig** [Rafiuth/Soggfy#46]
I have Soggify up and running, and it's brilliant, but I'm considering your warning that an account could get banned for using it. I suspect that Spotify would notice if someone were "playing" a track at 10x speed, but if playing at normal speed is there any way that Spotify could even guess that you are reconstructing the original file as you're playing it?

If there's really no way for Spotify to see anything out of the ordinary, when you are playing (and reconstructing) at normal speed, you might just update your Notes to reflect that,

Thanks,
Drew

**Comment by Rafiuth** [comment]
I agree with @SmilerRyan, it's highly unlikely that they can detect Soggfy currently, unless they add intrusive client-side checks or complicated server-side heuristics.

Adding on that, the normal Spotify client logs pretty much all player state changes: when a song starts or ends, volume, and yes, playback speed. Though these logs alone don't seem to trigger a ban, Soggfy blocks them by default via the "Misc > Block Telemetry" option. Because of that, I think it would be pretty unfeasible for them to implement a server-side detection heuristic based on just client behaviors (especially if you keep playback speed at 1x), downloaders just aren't that widespread for them to bother much.

It also seems that they use bans more like a scare tactic, saying that you can get unbanned by apologizing: https://www.reddit.com/r/Piracy/comments/x37e50/spotify_now_checking_for_piracy

**Comment by dtronvig** [comment]
Yeah, this all makes perfect sense.

Thanks @SmilerRyan and @Rafiuth.

**Comment by AlexandreGagner** [comment]
Hi,
I already had my main account banned using software that was implementing a similar technique (Sidify - look's like a Soggfy fork with UI that they sell). But the ban was after few month, like a manual batch lunched from Sportify.

Now I use Soggfy, but never on my main account. I just login using premium trial on alias email account, on LTE network and virtual VISA card that I delete after.

Also, I use always use full speed but I never for now received a ban message on my fake (alias) email account.

Bisous

**Comment by RealUlix** [comment]
Just found this tool and it works nicely. Very cool. I have questions: 
1. Has there been anyone banned by now because of it? 
2. I should be fine regardless, if I just leave the "Download button" crossed out on my main account, correct?
3. Recording on normal speed should (as far as anyone knows... no guarantees I know) be risk free?
